### PR TITLE
Remove My Deleted Repository

### PR DESCRIPTION
**Pull request by Developer-39**
Hello, my GitHub repository (https://github.com/Developer-39/Simple-QR-Code) has been deleted so I would like to remove it from this list.

Thank you in advance.

### CODE REVIEW

**Comment by per1234** [comment]
Hello @Developer-39. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Comment by Developer-39** [comment]
Sounds good, thank you for your assistance.

**Review by umbynos — APPROVED** [review]
Done:
```
2022/08/31 16:08:38 Loaded 5124 libraries from DB
Removing Simple QR Code
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/Developer-39/Simple-QR-Code not present
Success!
```
The warning should be expected since the library is no longer publicly available on GitHub

**Pull request merged**
